new_arrivals_chi/app/main.py

```diff
@@ -100,8 +106,16 @@ def signup():
 
 
 app = Flask(__name__)
-
 app.register_blueprint(main)
 
+app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv(
+    "DATABASE_URL", default="sqlite:///:memory:"
```

Comment:
It sets your default `DATABASE_URL` to the in-memory SQLlite if no other environment variable value is present.